protocol/proofs/generic-withdrawal-proofs.md

```diff
@@ -0,0 +1,88 @@
+# Super Root Withdrawal Proofs
+
+## Purpose
+
+Proposes a modification to the `OptimismPortal` that would add a new overload of
+`proveWithdrawalTransaction` that would allow users to prove against Super Roots.
+
+## Context
+
+Interop introduces the concept of the "Super Root" - a collection of Output Roots from all of the
+different chains in the interop set. The interop fault proof system argues over a super root rather
+than an individual Output Root. The `OptimismPortal` contract only supports proving withdrawals
+against a specific Output Root and will need to be modified to support the Super Root.
+
+## Problem Statement
+
+We need to modify the `OptimismPortal` contract to support proving withdrawals against a Super
+Root. Users would not be able to correctly execute withdrawals without this modification. This
+means that *some* solution to the problem is a basic requirement for interop.
+
+## Proposed Solution
+
+We propose to allow users to prove against Super Roots by adding a new overload of
+`proveWithdrawalTransaction` that allows a user to prove the existence of an Output Root inside of
+the Super Root before performing the standard withdrawal validation inside of the Output Root.
+
+We also specifically propose that this be implemented on the *base* `OptimismPortal` instead of
+introducing the code to the `OptimismPortalInterop` portal. Doing so means that we are including
+extra code on the `OptimismPortal` that is technically unnecessary until interop ships but also
+means that we do not need to ship any additional changes to the `OptimismPortal` before interop.
+
+We could expose a new public variable `usingSuperRoots` that would toggle this function on and off
+and would also act as a signal for client-side tooling to automatically switch between the two
+available functions. When `usingSuperRoots` is `false`, the version of `proveWithdrawalTransaction`
+for Super Roots would revert. When `usingSuperRoots` is `true`, the legacy version of the function
+would revert.
+
```

Comment:
We talked about this in the call, I'm not sure if it a decision was reached, but I do think it would reduce the amount of config air that is possible if we were to use the respected game type to set the value of usingSuperRoots. 

Other OK alternatives would be: 
1. a check in both `setUsingSuperRoots()` and `setRespectedGameType()` to ensure the values don't disagree.
2. adding a check to the StandardValidator contract